### DISCUSSION TITLE
Rework Spun Out to work with HR

### DISCUSSION THREAD

**Discussion by Zylviij** [ppy/osu#16837] [Ideas] [answered]
I think this may be an issue in stable osu too,
If you use spun out with hard rock, spinners can result in 100s, 50s, or even misses, causing health loss, acc loss, and combo breaks.

**Comment by peppy** [comment, marked as answer]
Tracking via https://github.com/ppy/osu/issues/16839.

**Reply by Zylviij** [reply]
Thank you!
